spec/models/article_spec.rb

```diff
@@ -3,5 +3,7 @@
 require 'rails_helper'
 
 RSpec.describe Article, type: :model do
-  # I will add tests with factory_bot
+  it { should validate_presence_of(:title) }
```

Comment:
"I'm still learning how to create factories and use them in RSpec. I'm going to add them in the next pr."
 Lets add them now? As i know we dont have much time so lets do it here and merge it into main so other participants will see how to use them.

Reply:
Let's do it, @olcplif ask me if you have questions about it

Reply:
@mmrshk  Tests added